src/main/resources/config/config.yaml

```diff
@@ -80,6 +80,10 @@ queueMessageReceiverConfig:
   numberOfThreads: ${QUEUE_MESSAGE_RECEIVER_NUMBER_OF_THREADS:-3}
   messageRetryDelayInSeconds: ${QUEUE_MESSAGE_RETRY_DELAY_IN_SECONDS:-900}
 
+webhookMessageDeletionConfig:
+  maxAgeOfMessages: ${MAXIMUM_AGE_OF_MESSAGE:-7}
+  maxNumOfMessagesToExpire: ${MAXIMUM_NUMBER_OF_MESSAGES_TO_EXPIRE_PER_JOB:-7500}
```

Comment:
The default config can be 15000 which will be a number of entries from the `webhook_messages` table. We can remove all associated entries from the queue table.